source/common/tcp_proxy/tcp_proxy.h

```diff
@@ -308,6 +310,9 @@ class Config {
   // This function must not be called if on demand is disabled.
   const OnDemandStats& onDemandStats() const { return shared_config_->onDemandConfig()->stats(); }
   Random::RandomGenerator& randomGenerator() { return random_generator_; }
+  const bool& flushAccessLogOnConnected() const {
```

Comment:
Same here, simple `bool` is better. nit: qualify the name to `OnUpstreamConnected`

Reply:
done